docs/static/spec/openapi/logstash-api.yaml

```
@@ -141,90 +289,913 @@ paths:
+      tags:
+        - node stats
+      parameters:
+        - $ref: "#/components/parameters/pretty"
+      responses:
+        "200":
+          description: A JSON object containing events statistics.
+          content:
+            application/json:
+              schema:
+                allOf:
+#                  - $ref: "#/components/schemas/Common"
+#                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/EventsStats"
+              example:
+                events:
+                  duration_in_millis: 1109
+                  in: 56
+                  filtered: 56
+                  out: 56
+                  queue_push_duration_in_millis: 2
+
+  /_node/stats/flow:
+    get:
+      summary: Get flow statistics
+      description: Get throughput and backpressure details.
+      operationId: nodeStatsFlow
+      tags:
+        - node stats
+      parameters:
+        - $ref: "#/components/parameters/pretty"
+      responses:
+        "200":
+          description: |
+            Flow rates provide visibility into how a Logstash instance or an individual pipeline is currently performing relative to itself over time. This allows us to attach meaning to the cumulative-value metrics that are also presented by this API, and to determine whether an instance or pipeline is behaving better or worse than it has in the past.
+
+            The following flow rates are available for the logstash process as a whole and for each of its pipelines individually. In addition, pipelines may have [additional flow rates](https://www.elastic.co/guide/en/logstash/current/node-stats-api.html#pipeline-flow-rates) depending on their configuration.
+          content:
+            application/json:
+              schema:
+                allOf:
+#                  - $ref: "#/components/schemas/Common"
+#                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/NodeStatsFlow"
+              example:
+                flow:
+                  input_throughput:
+                    current: 0.07381
+                    last_1_minute: 0.01572
+                    last_5_minutes: 0.01618
+                    last_15_minutes: 0.0173
+                    lifetime: 0.01701
+                  filter_throughput:
+                    current: 0.07379
+                    last_1_minute: 0.01572
+                    last_5_minutes: 0.01618
+                    last_15_minutes: 0.0173
+                    lifetime: 0.01701
+                  output_throughput:
+                    current: 0.07379
+                    last_1_minute: 0.01572
+                    last_5_minutes: 0.01618
+                    last_15_minutes: 0.0173
+                    lifetime: 0.01701
+                  queue_backpressure:
+                    "current": 0.00007379,
+                    "last_1_minute": 0.00001572,
+                    "last_5_minutes": 0.00004532,
```

The presence of these commas is the cause of this linting message:
`Example value must conform to the schema: `last_15_minutes` property must match exactly one schema in oneOf.`
```suggestion
                    "last_5_minutes": 0.00004532
```